src/Views/Filter/Subject.cshtml

```diff
@@ -76,7 +76,7 @@
                           @if (subject.IsSubjectKnowledgeEnhancementAvailable)
                             {
                               <p class="govuk-body">
-                                You can also take a <a href="https://getintoteaching.education.gov.uk/explore-my-options/teacher-training-routes/subject-knowledge-enhancement-ske-courses">subject knowledge enhancement (SKE) course</a>.
+                                You can@(subject.Funding?.Scholarship != null || subject.Funding?.BursaryFirst != null ? " also " : " ")take a <a href="https://getintoteaching.education.gov.uk/explore-my-options/teacher-training-routes/subject-knowledge-enhancement-ske-courses">subject knowledge enhancement (SKE) course</a>.
```

Comment:
is it a string can you use string.iswhitespace

Reply:
Actually it looks like an `int`

Reply:
HasValue

```csharp
subject.Funding?.Scholarship != null || subject.Funding?.BursaryFirst != null

subject.Funding?.Scholarship.HasValue || subject.Funding?.BursaryFirst.HasValue
```

Reply:
Throws `search-and-compare-ui/src/Views/Filter/Subject.cshtml(79,42): error CS0019: Operator '||' cannot be applied to operands of type 'bool?' and 'bool?' :/